pkg/hope/entities.go

```diff
@@ -74,11 +74,31 @@ type Job struct {
 
 // Node - Defines a networked resource on which operations will typically be
 //   executed.
+// Datastore is really only used for Hypervisors, but whatever; it's not
```

Comment:
Might eventually be worth getting a bit more sophisticated with this, and actually creating different types for these, and manually constructing them; unless there's another way.